.circleci/config.yml

```diff
@@ -49,14 +55,9 @@ jobs:
           keys:
             - gradle-{{ checksum "build.gradle.kts" }}
             - gradle-
-      - run:
-          command: |
-            ORG_GRADLE_PROJECT_signingKey=$(echo -e "$ORG_GRADLE_PROJECT_signingKey")
-            ./gradlew -Dorg.gradle.jvmargs="--illegal-access=permit" -Dkotlin.daemon.jvm.options="--illegal-access=permit" hmpps-kotlin-spring-boot-autoconfigure:publishToSonatype closeAndReleaseSonatypeStagingRepository
-      - run:
-          command: |
+      - run: |
             ORG_GRADLE_PROJECT_signingKey=$(echo -e "$ORG_GRADLE_PROJECT_signingKey")
-            ./gradlew -Dorg.gradle.jvmargs="--illegal-access=permit" -Dkotlin.daemon.jvm.options="--illegal-access=permit" hmpps-kotlin-spring-boot-starter:publishToSonatype closeAndReleaseSonatypeStagingRepository
+            ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
got four projects these days - hopefully this will publish all four (not sure why they were specified individually previously)